tests/vmcheck/test-kernel-args.sh

```diff
@@ -180,6 +180,22 @@ vm_rpmostree kargs --delete-if-present=PACKAGE3=TEST3 --unchanged-exit-77 || rc=
 assert_streq $rc 77
 echo "ok exit 77 when unchanged kargs with unchanged-exit-77"
 
+# Test for 'rpm-ostree kargs --editor'.
+vm_rpmostree kargs > kargs.txt
+assert_not_file_has_content_literal kargs.txt 'nonexisting'
+rc=0
+EDITOR="sed -i 's/nonexisting//'" vm_rpmostree kargs --editor --unchanged-exit-77 || rc=?
+assert_streq $rc 77
+vm_rpmostree kargs > kargs.txt
+assert_not_file_has_content_literal kargs.txt 'nonexisting'
+vm_rpmostree kargs > kargs.txt
+assert_not_file_has_content_literal kargs.txt 'editorkey=editorvalue'
+rc=0
+EDITOR="sed -i 's/$/ editorkey=editorvalue/'" vm_rpmostree kargs --editor --unchanged-exit-77
+assert_streq $rc 0
+vm_rpmostree kargs > kargs.txt
+assert_file_has_content_literal kargs.txt 'editorkey=editorvalue'
```

Comment:
For completeness, would be good to add another test here to check that `--editor` without `--unchanged-exit-77` returns an error if nothing changed.